src/nvim/fileio.c

```diff
@@ -366,14 +366,13 @@ readfile (
   /* don't display the file info for another buffer now */
   need_fileinfo = FALSE;
 
-  /*
-   * For Unix: Use the short file name whenever possible.
-   * Avoids problems with networks and when directory names are changed.
-   * Don't do this for MS-DOS, a "cd" in a sub-shell may have moved us to
```

Comment:
:+1: we still need `BACKSLASH_IN_FILENAME` defined somewhere though I think. I couldn't find it in `master` or in a commit in #810.

But that's for another PR.